src/signals/incident/components/form/RadioInput/InputUsingDispatch.js

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback } from 'react';
```

Comment:
could we move this in a ./components/InputUsingDispatch folder according to the convention?